### DISCUSSION TITLE
Severity computation

### DISCUSSION THREAD

**Discussion by ataumo** [0x6d69636b/windows_hardening#73] [Q&A] [answered]
How do you exactly compute severity ? For example, in CIS Benchmarks you have only 2 levels : L1, L2. So, how can you add a level ? Is it based on these levels or it's only your own estimation ? 

Also, what is the definition (exactly) of High, Medium and Low ?

**Comment by 0x6d69636b** [comment, marked as answer]
Level 1 and 2 are specifications from CIS, I have simply adopted them. Therefore, I will not make any adjustments or add more levels there.

The severity levels are my personal assessments if the framework does not give any guidelines. In the case of NIST/STIG, they have defined severity levels themselves. Basically, I use [CVSS](https://www.first.org/cvss/v3.1/specification-document#Qualitative-Severity-Rating-Scale) as a guideline, but this is not quite as easy to convert for configuration reviews. Since a missing configuration setting does not directly represent a vulnerability.

**Reply by ataumo** [reply]
Ok ! Thank you ! :)
